.rubocop.yml

```diff
@@ -39,4 +39,38 @@ Metrics/BlockLength:
     - it
     - class_eval
     - has_metadata
-    - resources
\ No newline at end of file
+    - resources
+
+AllCops:
+  Exclude:
+    - bin/*
+    - vendor/**/*
```

Comment:
Exclude the cops from the vendor directory, otherwise things get very *shouty* on travis
We also exclude the bin folder as this is mostly generated by bundler/rails etc and results in noise.